docs/source/reference/multi_label_classification.rst

```diff
@@ -0,0 +1,216 @@
+
+.. _multi_label_classification:
+
+################################
+Multi-label Image Classification
+################################
+
+********
+The task
+********
+Multi-label classification is the task of assigning a number of labels from a fixed set to each data point, which can be in any modality. In this example, we will look at the task of trying to predict the movie genres from an image of the movie poster.
+
+------
+
+********
+The data
+********
+The data we will use in this example is a subset of the awesome movie poster genre prediction data set from the paper "Movie Genre Classification based on Poster Images with Deep Neural Networks" by Wei-Ta Chu and Hung-Jui Guo, resized to 128 by 128.
+Take a look at their paper (and please consider citing their paper if you use the data) here: `www.cs.ccu.edu.tw/~wtchu/projects/MoviePoster/ <https://www.cs.ccu.edu.tw/~wtchu/projects/MoviePoster/>`_.
+
+------
+
+*********
+Inference
+*********
+
+The :class:`~flash.vision.ImageClassifier` is already pre-trained on `ImageNet <http://www.image-net.org/>`_, a dataset of over 14 million images.
+
+We can use the :class:`~flash.vision.ImageClassifier` model (pretrained on our data) for inference on any string sequence using :func:`~flash.vision.ImageClassifier.predict`.
+We can also add a simple visualisation by extending :class:`~flash.data.base_viz.BaseVisualization`, like this:
+
+.. code-block:: python
+
+    # import our libraries
+    from typing import Any
+
+    import torchvision.transforms.functional as T
+    from torchvision.utils import make_grid
+
+    from flash import Trainer
+    from flash.data.base_viz import BaseVisualization
+    from flash.data.utils import download_data
+    from flash.vision import ImageClassificationData, ImageClassifier
+    from flash.vision.classification.data import ImageClassificationPreprocess
+
+    # 1. Download the data
+    download_data("https://pl-flash-data.s3.amazonaws.com/movie_posters.zip", "data/")
+
+    # 2. Define our custom visualisation and datamodule
+    class CustomViz(BaseVisualization):
+
+        def show_per_batch_transform(self, batch: Any, _):
+            images = batch[0]
+            image = make_grid(images, nrow=2)
+            image = T.to_pil_image(image, 'RGB')
+            image.show()
+
+
+    class CustomMultiLabelPreprocess(ImageClassificationPreprocess):
```

Comment:
Why it this needed ?

Reply:
The class is created in fine-tuning example and doesn't exist in the predict one. Needs to be added.